ElementX/Sources/Services/Client/ClientProxy.swift

```diff
@@ -105,7 +105,8 @@ class ClientProxy: ClientProxyProtocol {
 
         let delegate = WeakClientProxyWrapper(clientProxy: self)
         client.setDelegate(delegate: delegate)
-        // Uncomment to test local notifications
+
+        // Comment to remove local notifications
```

Comment:
Possibly more relevant?
```suggestion
        // Set up sync listener for generating local notifications.
```